internal/pkg/service/common/httpserver/middleware/otel.go

```diff
@@ -87,14 +81,6 @@ func otelOptions(cfg otelConfig, tp trace.TracerProvider, mp metric.MeterProvide
 	return out
 }
 
-func metricAttrs(req *http.Request) (out []attribute.KeyValue) {
-	// Route
-	if routerData := httptreemux.ContextData(req.Context()); routerData != nil {
-		out = append(out, semconv.HTTPRoute(routerData.Route()))
-	}
-	return out
-}
-
 func spanRequestAttrs(cfg *otelConfig, req *http.Request) (out []attribute.KeyValue) {
```

Comment:
Toto bolo duplicitne, riesi sa v `otel_route`: https://github.com/keboola/keboola-as-code/blob/fe9835732edd57f359361e9cc9e9f82c9256c691/internal/pkg/service/common/httpserver/middleware/otel_route.go#L33 (tu to ani nefungovalo, middleware sa musi zaregistrovat do `httptreemux`)